app/uk/gov/hmrc/overseaspensiontransferbackend/models/SchemeManagerDetails.scala

```diff
@@ -16,16 +16,20 @@
 
 package uk.gov.hmrc.overseaspensiontransferbackend.models
 
+import play.api.libs.functional.syntax.toFunctionalBuilderOps
 import play.api.libs.json._
 
 case class SchemeManagerDetails(
-    schemeManagerType: Option[String]
+    schemeManagerType: Option[String],
+    schemeManagerAddress: Option[Address]
```

Comment:
There is a SchemeManagerAddress type in AddressTypes, in this case it isn't strictly necessary but you know what the API spec is like